articles/access_controls.md

```diff
@@ -1,58 +1,57 @@
 ---
-title: "Access and Content Restrictions"
+title: "Access Controls and Data Governance: Sharing Settings and Conditions for Use"
 layout: article
-excerpt: Learn how to share and restrict content on Synapse using the sharing settings and conditions for use. 
+excerpt: Learn how to set access controls and data governance on Synapse using `Sharing settings` and `Conditions for Use`. 
 ---
```

Comment:
I am hoping to move all of us to one set of descriptors for one set of
terms. Because our technical team uses "access controls" consistently and
because "access control" does describe what "Sharing settings" do, I went
with that term. Is there a different term that you feel is more universally
used at Sage to describe "Sharing settings"?

Yooree and I have touched base on formatting.

On Thu, Jun 30, 2016 at 2:53 PM, Larsson Omberg <notifications@github.com>
wrote:

> In articles/access_controls.md
> <https://github.com/Sage-Bionetworks/synapseDocs/pull/118#discussion_r69217285>
> :
>
> >  layout: article
> > -excerpt: Learn how to share and restrict content on Synapse using the sharing settings and conditions for use.
> > +excerpt: Learn how to set access controls and data governance on Synapse using `Sharing settings` and `Conditions for Use`.
> >  ---
>
> Access Controls seems like a more technical term than share and restrict
> content? Also for the summary we shouldn't use the backtick notation as it
> would visually different in the summary.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Sage-Bionetworks/synapseDocs/pull/118/files/4028eff245c3695581c8d12c4f9b5abf173e0306#r69217285>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQya_n-blV52hbFXYRE9v7gIixV_hrnpks5qRDrZgaJpZM4JChJF>
> .
>



-- 
Megan Doerr, MS, LGC
Principal Scientist, Governance
Sage Bionetworks
phone: (206) 667-4265
1100 Fairview Ave N.
M1-C108
Seattle, WA 98109